source/_integrations/fritzbox.markdown

```diff
@@ -33,7 +33,7 @@ There is currently support for the following device types within Home Assistant:
 
 ## Configuration
 
-To add the AVM FRITZ!Box integration to your installation, go to **Configuration** -> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **AVM FRITZ!Box**.
+To add the AVM FRITZ!SmartHome integration (which is a part of the AVM FRITZ!Box integration) to your installation, go to **Configuration** -> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **AVM FRITZ!Box**.
```

Comment:
It was so nobody gets confused, because in the integrations tab, you add the "AVM FRITZ!Box" Integration, mit the "AVM FRITZ!SmartHome" Integration. I can remove it if that causes even more confusion.

Reply:
Changed it now.